DayCare/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class DayCare(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, blank=True, null=True)
+    name = models.CharField(max_length=20)
+    description = models.TextField(blank=True, null=True)
+    price_per_day = models.DecimalField(decimal_places=2, max_digits=10000, blank=False, null=False, default=0)
+    capacity = models.IntegerField()
+    area = models.CharField(max_length=20)
+    city = models.CharField(max_length=30)
+    address = models.CharField(max_length=100)
+
+    @receiver(post_save, sender=User)
```

Comment:
Why does is it necessary to use the @receiver annotation, sender, instance, create all this magics?

Reply:
Fixed the class model